src/utils/cache/BalanceCache.ts

```diff
@@ -47,4 +60,24 @@ export class BalanceCache extends EntityCache<string, BalancesResponse | null> {
         }
         return result
     }
+
+
+    /**
+     * @notice The `api/v1/accounts/${accountId}/tokens` endpoint can return a maximum of 100 records. 
+     *         If more records are available, a `links.next` endpoint will be returned in the response. 
+     *         Make a new request at `links.next` endpoint to fetch the next batch of records.
+     * 
+     * @dev Recursively make request to mirror node using the `tokensReponse.links.next` url to fetch all records
+     */
+    private requestNextTokenBatch: any = async (tokensReponse: TokenResponse, tokens: TokenBalance[]) => {
+        if (!tokensReponse.links.next) {
+            tokens.push(...tokensReponse.tokens)
```

Comment:
74 and 78 rows are duplicate.
Can be simplified by doing .push() before if.

Reply:
Ah you're right! Thanks for pointing it out @moshenskyDV!